src/macrofier/types.mjs

```diff
@@ -137,6 +139,39 @@ const insertEnumMacros = (content, schema, module, name) => {
   return template.join('\n')
 }
 
+const insertObjectAdditionalPropertiesMacros = (content, schema, module, title, options) => {
+  options = options ? JSON.parse(JSON.stringify(options)) : {}
+
+  const options2 = JSON.parse(JSON.stringify(options))
+  options2.parent = title
```

Comment:
You've already made a copy on line 143. Why do you need a second copy?

Reply:
Also, I can't find where this new `insertObjectAdditionalProperitesMacros` method is called.

Reply:
sorry missed to updated calling place. corrected now